app/main.py

```
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.strip().split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    src, dst = parts[1], parts[2]
+
+    if src == dst:
+        return
+
+    try:
+        with open(src, "rb") as file_in, open(dst, "wb") as file_out:
+            file_out.write(file_in.read())
+    except FileNotFoundError:
```

The requirements do not specify that you should catch `FileNotFoundError` or handle exceptions. The function should not swallow errors unless explicitly required. This violates the implementation requirements.